app/views/govuk_publishing_components/components/_cookie_banner.html.erb

```diff
@@ -1,72 +1,80 @@
 <%
   id ||= 'global-cookie-message'
-  title ||= "Tell us whether you accept cookies"
-  text ||= raw("We use <a class='govuk-link' href='/help/cookies'>cookies to collect information</a> about how you use GOV.UK. We use this information to make the website work as well as possible and improve government services.")
+  title ||= "Cookies on GOV.UK"
+
+  text ||= raw("<p>We use some essential cookies to make this website work.</p>
+                <p>We’d like to set additional cookies so we and our partners can remember your settings, understand how you use GOV.UK and make improvements.</p>
```

Comment:
I'm wondering if there's a way to stop this being a breaking change. Could it detect if it's passed a single string (the old way) and wrap that in a para and render it, but otherwise expect an array of strings, which it would then each wrap in a para and render?

This is a bit messy, but it seems to work.

```
  text ||= ["We use some essential cookies to make this website work", "We'd like to set additional cookies to understand how you use GOV.UK, remember your settings and improve government services.", "We also use cookies set by other sites to help us deliver content from their services."]

  if text.kind_of?(Array)
    newtext = ""
    text.each do |t|
      newtext += "<p>#{t}</p>"
    end

    text = newtext
  else
    text = "<p>#{text}</p>"
  end

  text = raw(text)
```

Reply:
This would also allow us to insert the `govuk-body` class on the `p` elements, and remove lines 71-74 of the stylesheet, where we go against our coding standards and style a `p` element without a class.

Reply:
Done. PR and changelog updated accordingly.

Reply:
Sadly the overrides are still needed because govuk-template has a selector with an ID that overrides this and shrinks all the text.